src/main/resources/FXML/Menu.fxml

```diff
@@ -39,14 +55,14 @@
                     <items>
                       <AnchorPane minHeight="-Infinity" minWidth="-Infinity" prefWidth="150.0">
                            <children>
-                              <TextField fx:id="nodeTextField" alignment="CENTER" layoutX="27.0" layoutY="30.0" prefHeight="29.0" prefWidth="97.0" promptText="Centre node" />
-                              <Button layoutX="14.0" layoutY="71.0" mnemonicParsing="false" onAction="#traverseGraphClicked" text="Go to this node" />
-                              <TextField fx:id="radiusTextField" alignment="CENTER" layoutX="27.0" layoutY="111.0" prefHeight="29.0" prefWidth="97.0" promptText="Radius" />
-                              <Button fx:id="saveBookmark" layoutX="15.0" layoutY="224.0" mnemonicParsing="false" onAction="#saveTheBookmarks" text="Save Bookmark" />
-                              <Button fx:id="bookmark1" layoutX="27.0" layoutY="268.0" mnemonicParsing="false" onAction="#pressBookmark1" prefHeight="24.0" prefWidth="98.0" text="-" textAlignment="CENTER" />
-                              <Button fx:id="bookmark2" layoutX="27.0" layoutY="311.0" mnemonicParsing="false" onAction="#pressBookmark2" prefHeight="24.0" prefWidth="97.0" text="-" textAlignment="CENTER" />
-                              <CheckBox fx:id="dummyNodeCheckbox" layoutX="62.0" layoutY="405.0" mnemonicParsing="false" onAction="#toggleDummyNodes" />
-                              <Label layoutX="3.0" layoutY="372.0" text="Display dummy nodes" />
+                              <TextField fx:id="nodeTextField" alignment="CENTER" layoutX="27.0" layoutY="30.0" prefHeight="29.0" prefWidth="97.0" promptText="Centre node" visible="false" />
```

Comment:
I agree, but originally that caused an odd bug where the menu bar with file-help etc. would stop working. I will look into it